beacon-chain/rpc/eth/light-client/helpers.go

```diff
@@ -19,13 +18,8 @@ func newLightClientFinalityUpdateFromBeaconState(
 	attestedState state.BeaconState,
 	attestedBlock interfaces.ReadOnlySignedBeaconBlock,
 	finalizedBlock interfaces.ReadOnlySignedBeaconBlock,
-) (*structs.LightClientFinalityUpdate, error) {
-	result, err := lightclient.NewLightClientFinalityUpdateFromBeaconState(ctx, currentSlot, state, block, attestedState, attestedBlock, finalizedBlock)
-	if err != nil {
-		return nil, err
-	}
-
-	return structs.LightClientFinalityUpdateFromConsensus(result)
+) (interfaces.LightClientFinalityUpdate, error) {
+	return lightclient.NewLightClientFinalityUpdateFromBeaconState(ctx, currentSlot, state, block, attestedState, attestedBlock, finalizedBlock)
```

Comment:
We can call this function directly from handlers, not need to go through the helper function that doesn't do anything.